tests/integration/__tests__/patterns-dirs.js

```diff
@@ -153,7 +153,24 @@ if (path.sep === "/") {
   });
 }
 
-describe("Ignore symlinks", () => {
+function isSymlinkSupported() {
+  if (process.platform !== "win32") {
+    return true;
+  }
+
+  const target = path.join(__dirname, "../cli/patterns-symlinks");
+  const symlink = path.join(__dirname, "../cli/patterns-symlinks/symlink");
```

Comment:
Let's link to a file inside to test, prevent accidentally removes the parent dir.

Reply:
`unlinkSync` can never delete a directory, but sure, changed.